akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -8,20 +8,53 @@
 
 //#log
 import akka.stream.Attributes;
+import akka.stream.javadsl.Source;
+
+import java.util.Arrays;
```

Comment:
those imports are not part of the log example

Reply:
good catch, thanks